config/.env.example

```
@@ -12,7 +12,7 @@ API_GQL_URL=http://localhost:17608/query
 NEXT_PUBLIC_OPENLANE_URL=http://localhost:17608
 NEXT_PUBLIC_API_GQL_URL=http://localhost:17608/query
 SESSION_COOKIE_NAME=temporary-cookie
-SESSION_COOKIE_EXPIRATION_MINUTES=10
+SESSION_COOKIE_EXPIRATION_MINUTES=1
```

```suggestion
SESSION_COOKIE_EXPIRATION_MINUTES=10
```

^ we should put this back so locally we aren't having to re-login after 1 minute